### PR TITLE
Feature/captcha toggle

### PR DESCRIPTION
**Pull request by stuaxo**

### CODE REVIEW

File: contact/views.py [hunk 1]
```diff
@@ -44,6 +50,7 @@ def create_zendesk_ticket(self, cleaned_data, zendesk_user):
         zenpy_client.tickets.create(ticket)
 
     def form_valid(self, form):
-        zendesk_user = self.get_or_create_zendesk_user(form.cleaned_data)
-        self.create_zendesk_ticket(form.cleaned_data, zendesk_user)
+        if settings.ENABLE_CAPTCHA:
```
Comment (konradko) [inline contact/views.py]: so if captcha is disabled you don't post to zendesk?

File: contact/templates/report_issue-form.html [hunk 2]
```diff
@@ -47,7 +47,9 @@ <h2 class="element-within title">
 				<div class="row-fluid">
 					<div class="span12">
 						<div class='input-field-container'>
-							{{ form.captcha }}
+              {% if ENABLE_CAPTCHA %}
```
Comment (konradko) [inline contact/templates/report_issue-form.html]: this will just not display the field, it will still be required server-side

**Comment by konradko** [comment]
@stuaxo 👍 to what @richtier suggested. But if the purpose is to remove the captcha the change has to be in https://github.com/uktrade/invest/blob/master/contact/forms.py#L9

**Comment by stuaxo** [comment]
Yep - definitely doing something silly here...

Basically, I want to not do most of the above -

Instead just make the recpatcha field  ```required = settings.ENABLE_CAPTCHA```  ?

**Comment by stuaxo** [comment]
Ah, think I added the code to disable zendesk when in an attempt to make the form valid.

**Comment by stuaxo** [comment]
@richtier using the testing config makes the most sense to me, I'm not sure why pentesters wants to remove it entirely.

**Comment by stuaxo** [comment]
@richtier using the testing config makes the most sense to me, I'm not sure why pentesters suggested removing it entirely.

**Comment by stuaxo** [comment]
They gave the OK, for using testing - so I get to delete this branch :)

**Comment by stuaxo** [comment]
So, I somehow undeleted this :/

**Pull request closed without merging**